package.json

```diff
@@ -13,7 +13,8 @@
     },
     "homepage": "https://github.com/fluid-lab/gamepad-navigator#readme",
     "scripts": {
-        "lint": "grunt lint"
+        "lint": "grunt lint",
+        "test": "testem ci --file tests/testem.json"
```

Comment:
You need to add testem as a dev dependency so that your tests can be run with a particular version of testem both locally and in CI.